src/main/scala/nl.knaw.dans.easy.license/LicenseCreatorApp.scala

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.license
+
+import java.io.File
+import javax.naming.Context
+import javax.naming.ldap.InitialLdapContext
+
+import com.yourmediashelf.fedora.client.{ FedoraClient, FedoraCredentials }
+import rx.schedulers.Schedulers
+
+class LicenseCreatorApp(configuration: Configuration) extends AutoCloseable {
```

Comment:
It looks like this is supposed to go in `LicenseCreatorWiring`, right? The `App` was to provide a public API for the web-API and the cmd-API